ffi/passmanagers.cpp

```diff
@@ -475,9 +475,7 @@ LLVMPY_AddBasicAliasAnalysisPass(LLVMPassManagerRef PM) {
 }
 
 API_EXPORT(void)
-LLVMPY_LLVMAddLoopRotatePass(LLVMPassManagerRef PM) {
-    LLVMAddLoopRotatePass(PM);
-}
+LLVMPY_AddLoopRotatePass(LLVMPassManagerRef PM) { LLVMAddLoopRotatePass(PM); }
```

Comment:
Note to other reviewers - there was a slight inconsistency in the naming here, with the redundant `LLVM` in the name, which was removed as part of a bit of general clean-up (not that it matters greatly, as this code will eventually be removed).